src/components/Icon/CreditsIcon.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+import SvgIcon, { SvgIconProps } from "@mui/material/SvgIcon"
+
+const CreditsIcon = React.memo((props: SvgIconProps) => {
+  return (
+    <SvgIcon {...props}>
+      <svg
+        width="12"
+        height="15"
+        viewBox="0 0 12 15"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <path
+          fillRule="evenodd"
+          clipRule="evenodd"
+          d="M1.33705 4.16563C1.38647 3.82008 1.68242 3.56344 2.03148 3.56344L9.96852 3.56344C10.3176 3.56344 10.6135 3.82008 10.6629 4.16563L11.9929 13.4653C12.0217 13.6667 11.9617 13.8706 11.8285 14.0242C11.6952 14.1779 11.5019 14.2661 11.2985 14.2661H0.701503C0.498126 14.2661 0.304759 14.1779 0.171518 14.0242C0.0382768 13.8706 -0.0217225 13.6667 0.0070702 13.4653L1.33705 4.16563ZM2.6398 4.96644L1.51046 12.8631H10.4895L9.3602 4.96644H2.6398Z"
+          fill="#A0ABFF"
+        />
+        <path
+          fillRule="evenodd"
+          clipRule="evenodd"
+          d="M3.64364 3.19351C3.64364 1.892 4.69872 0.836914 6.00023 0.836914C7.30174 0.836914 8.35682 1.892 8.35682 3.1935V7.36617C8.35682 7.7536 8.04275 8.06767 7.65532 8.06767C7.26789 8.06767 6.95382 7.7536 6.95382 7.36617V3.1935C6.95382 2.66685 6.52688 2.23991 6.00023 2.23991C5.47358 2.23991 5.04664 2.66685 5.04664 3.19351V7.36617C5.04664 7.7536 4.73256 8.06767 4.34514 8.06767C3.95771 8.06767 3.64364 7.7536 3.64364 7.36617V3.19351Z"
+          fill="#A0ABFF"
```

Comment:
is this ok in dark/light theme?

Reply:
<img width="522" alt="Screenshot 2025-03-25 at 9 53 46 AM" src="https://github.com/user-attachments/assets/1fe11c2b-ec9e-44c0-9692-13b9f1ef1b3d" />
<img width="491" alt="Screenshot 2025-03-25 at 9 54 06 AM" src="https://github.com/user-attachments/assets/5b978d09-369f-4a66-93af-598c11346dbd" />